internal/api/campaign_run.go

```diff
@@ -35,6 +36,8 @@ type Campaign struct {
 // run name is used to identify the campaign run in the API for consecutive
 // calls.
 func (client *APIClient) CreateCampaignRun(project string, token string, fuzzTarget string, numBuildJobs uint) (string, string, error) {
+	fuzzTarget = base64.StdEncoding.EncodeToString([]byte(fuzzTarget))
```

Comment:
```suggestion
	fuzzTarget = base64.URLEncoding.EncodeToString([]byte(fuzzTarget))
```

We use the base 64 URL Encoding for fuzz target names.